main.go

```diff
@@ -53,9 +53,11 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.Parse()
 
-	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
+	opts := zap.Options{}
+	opts.BindFlags(flag.CommandLine)
+	flag.Parse()
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
```

Comment:
is this actually parsing all options to look for a `zap-devel`?
For some reason I can't find a doc about `UseFlagOptions`

Reply:
It seems to me that is looking at all the flags and using only the zap ones

The docs are at
https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/log/zap@v0.8.0#UseFlagOptions